core/handler_schema_constants.py

```diff
@@ -54,6 +54,8 @@
     'PendingAccountDeletionPage',
     'PreferenceHandler',
     'PreferencesHandler',
+    'ProfilePictureDataUrlHandler',
+    'EmailPreferencesHandler',
```

Comment:
Don't add these here rather immediately add the schemas

Reply:
okay, will do that.

Reply:
Added schema for those handler classes